clio-common/src/main/java/eu/europeana/clio/common/model/BatchWithCounters.java

```diff
@@ -0,0 +1,63 @@
+package eu.europeana.clio.common.model;
+
+import java.time.Instant;
+
+/**
+ * This class represents a batch with some extra counters.
+ */
+public class BatchWithCounters {
+
+  private final long batchId;
+  private final Instant creationTime;
+  private final Instant lastUpdateTimeInSolr;
+  private final Instant lastUpdateTimeInMetsiCore;
```

Comment:
Spelling mistake here, same for methods etc..

Reply:
Oops...